static_dns/dns.tf

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Should this file be named `records.tf` maybe?

Reply:
make sense